tldw_Server_API/app/core/Usage/audio_quota.py

```
@@ -417,6 +647,27 @@ async def finish_job(user_id: int) -> None:
     """
+    # When RG audio integration is enabled, release one jobs concurrency lease
+    # for this user if present, then fall back to legacy counters for metrics.
+    gov = await _get_audio_rg_governor()
+    if gov is not None:
+        try:
+            handles = _rg_job_handles.get(int(user_id)) or []
+            if handles:
+                handle_id = handles.pop()
+                try:
+                    await gov.release(handle_id)
+                except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)